configs/rfnext/README.md

```diff
@@ -0,0 +1,177 @@
+# RF-Next: Efficient Receptive Field Search for CNN
```

Comment:
Please add a `metafile.yml` for rfnext just like https://github.com/open-mmlab/mmdetection/blob/master/configs/solov2/metafile.yml

Reply:
I have added the `metafile.yaml`.